frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java

```diff
@@ -73,15 +74,14 @@
 
     public WorkerThread(
             ConfigManager configManager,
-            List<WorkerThread> parentThreads,
             EventLoopGroup backendEventGroup,
             int port,
             int gpuId,
             Model model,
             BatchAggregator aggregator,
-            WorkerStateListener listener) {
+            WorkerStateListener listener,
+            int backoffIdx) {
```

Comment:
You don't need pass int backoffIdx since you are reusing WorkerThread object.
But you need reset it on success load model.

Reply:
Good point. I need to reset it. I will do that.